.vscode/extension.code-snippets

```diff
@@ -0,0 +1,41 @@
+{
+  "New unsandboxed TurboWarp extension": {
+    "prefix": "~extension",
+    "description": "A template for an official TurboWarp extension.",
+    "isFileTemplate": true,
+    "scope": "javascript",
+    "body": [
+      "// Name: ${1:My Extension}",
+      "// ID: ${2:myextension}",
+      "// Description: $3",
+      "// By: ${4:me} <https://scratch.mit.edu/users/$4/>",
+      "// License: ${5|Choose,MPL-2.0,MIT,Apache-2.0,LGPL-3.0|}",
```

Comment:
IMO maybe this could just be a choice between MPL-2.0 and MIT, or even just MPL-2.0